### PR TITLE
[15.0][Add] product_sale_manufactured_for

### PR DESCRIPTION
**Pull request by santostelmo**
Add a new module that allows to indicate in products that they were made
(i.e. manufactured) specifically for some customers.

In a sale order, allow to only select products from
the UI that were manufactured for the chosen customer.

Depends of sale_commercial_partner: https://github.com/OCA/sale-workflow/pull/1747

### CODE REVIEW

**Review by ajaniszewska-dev — APPROVED** [review]
LG

File: product_sale_manufactured_for/README.rst [hunk 1]
```diff
@@ -0,0 +1,87 @@
+=======================
+Sale Commercial Partner
```
Comment (ajaniszewska-dev) [inline product_sale_manufactured_for/README.rst]: are you sure it is correct module name here?

Comment (santostelmo) [reply product_sale_manufactured_for/README.rst]: No is wrong. The name should be "Product Manufactured for Customer"

**Comment by rousseldenis** [comment]
@santostelmo Could you fix pre-commit and squash your commits ?

**Pull request closed without merging**